src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -1661,9 +1628,9 @@ def test_ps_s3_multipart_on_master():
         key.delete()
     # delete the bucket
     conn.delete_bucket(bucket_name)
-    clean_rabbitmq(proc)
 
 
+@attr('Still not AMQP test')
```

Comment:
for this test to work, you need to install `service-2.sdk-extras.json` on the test machine. see: https://github.com/ceph/ceph/blob/master/examples/boto3/README.md